packages/x-tailwindcss/demo/src/components/xds-layout.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <XdsBaseShowcase
+    #default="{ cssClass, copyCssClassesToClipboard, removeClassPrefix }"
+    title="Layout"
+    :sections="sections"
+  >
+    {{ removeClassPrefix(cssClass, base) }}
+    <div
+      :key="cssClass"
+      @click="copyCssClassesToClipboard"
+      @keydown="copyCssClassesToClipboard"
+      :class="cssClass"
+      title="Click me to copy CSS classes"
+      class="x-bg-neutral-50"
+    >
+      <div v-for="index in 5" :key="index" class="x-layout-item">
+        {{ index }}
+      </div>
+    </div>
+  </XdsBaseShowcase>
+</template>
+
+<script lang="ts">
+  import { Vue, Component, Prop } from 'vue-property-decorator';
+  import { ShowcaseSections } from '../types/types';
+  import { addParentClasses } from '../utils';
+  import XdsBaseShowcase from './xds-base-showcase.vue';
+
+  @Component({
+    components: {
+      XdsBaseShowcase
+    }
+  })
+  export default class XdsLayoutShowcase extends Vue {
+    @Prop({ default: 'x-layout-container' })
+    public base!: string;
+
+    @Prop({
+      default: () => [
+        'x-layout-container-sm',
+        'x-layout-container-md',
+        'x-layout-container-lg',
+        'x-layout-container-full'
+      ]
+    })
+    public sizes!: string[];
+
+    @Prop({
+      default: () => [
+        'x-layout-container-min-margin-12',
+        'x-layout-container-min-margin-20',
+        'x-layout-container-min-margin-32',
+        'x-layout-container-min-margin-48'
```

Comment:
Maybe it can lead to confusion to only have 4 classes listed here instead of all the available (as many as spacing values in the theme) but I don't see a better approach for now.

Reply:
It would be too much, and this showcase's goal should be focused on the relationship between the items and the container. Maybe we can add a comment in the showcase file to point out that we can use all the margins available in the `theme.ts` file. What do you think @mnavarroespinosa?

Reply:
Reading the task, it said that maybe we should follow the approach of creating a template like the archetype one:
```
The showcase can be similar to the layout used in the archetype. It can be started in this task and completed in the layout item task.
```

So maybe we should remove this showcase and create one that simulates a real layout.